### ISSUE TITLE
Change coding style to black

### ISSUE THREAD

**Issue by gagan3012** [gradsflow/gradsflow#16]
#### Is your feature request related to a problem? Please describe.
I think it would be better to implement black (https://github.com/psf/black) as a coding style for Gradsflow.

**Comment by aniketmaurya** [comment]
yes, the current `pre-commit` hook formats code with Black formatter - https://github.com/gradsflow/gradsflow/blob/main/.pre-commit-config.yaml#L11
